testing/dependencies/appveyor/test.bat

```diff
@@ -2,14 +2,19 @@ rem install pipenv dependencies
 bash -lc "cd /c/projects/cmake-cookbook-no-symlinks && pipenv install"
 
 rem finally we start with the actual tests
-bash -lc "cd /c/projects/cmake-cookbook-no-symlinks && pwd && pipenv run python testing/collect_tests.py 'chapter-11/recipe-03'"
-bash -lc "cd /c/projects/cmake-cookbook-no-symlinks && pwd && pipenv run python testing/collect_tests.py 'chapter-11/recipe-02'"
-bash -lc "cd /c/projects/cmake-cookbook-no-symlinks && pwd && pipenv run python testing/collect_tests.py 'chapter-11/recipe-01'"
-bash -lc "cd /c/projects/cmake-cookbook-no-symlinks && pwd && pipenv run python testing/collect_tests.py 'chapter-09/recipe-*'"
-bash -lc "cd /c/projects/cmake-cookbook-no-symlinks && pwd && pipenv run python testing/collect_tests.py 'chapter-07/recipe-*'"
-bash -lc "cd /c/projects/cmake-cookbook-no-symlinks && pwd && pipenv run python testing/collect_tests.py 'chapter-06/recipe-*'"
-bash -lc "cd /c/projects/cmake-cookbook-no-symlinks && pwd && pipenv run python testing/collect_tests.py 'chapter-05/recipe-*'"
-bash -lc "cd /c/projects/cmake-cookbook-no-symlinks && pwd && pipenv run python testing/collect_tests.py 'chapter-04/recipe-*'"
-bash -lc "cd /c/projects/cmake-cookbook-no-symlinks && pwd && pipenv run python testing/collect_tests.py 'chapter-03/recipe-*'"
-bash -lc "cd /c/projects/cmake-cookbook-no-symlinks && pwd && pipenv run python testing/collect_tests.py 'chapter-02/recipe-*'"
-bash -lc "cd /c/projects/cmake-cookbook-no-symlinks && pwd && pipenv run python testing/collect_tests.py 'chapter-01/recipe-*'"
+if "%ANACONDA_TESTS_ONLY%"=="1" (
+  bash -lc "cd /c/projects/cmake-cookbook-no-symlinks && pwd && pipenv run python testing/collect_tests.py 'chapter-11/recipe-04'"
```

Comment:
Is the `-lc` flag needed? It will start a new login shell and clean up the `PATH` (I think)

Reply:
I am not sure. I asked myself the same question when migrating to MSYS2. I can investigate on Azure. When starting with MSYS2 I browsed a number of other projects on GitHub to see how they do it and this was the common approach but as you say it forgets "everything" (not sure what persists) between calls.

Reply:
I'll open an issue because I am not particularly hot on debugging this right now :smile_cat:

Reply:
Good plan.